class Exception2{
   static int sum(int num1, int num2){
      if (num1 == 0)
         throw new ArithmeticException("First parameter is not valid");
      else
         System.out.println("Both parameters are correct!!");
      return num1+num2;
   }
   public static void main(String args[]){
      int res=sum(0,12);
      System.out.println(res);
      System.out.println("Continue Next statements");
   }
}


use of throw & throws


public class complete {
    
  void mymethod(int num)throws IOException, ClassNotFoundException{ 
     if(num==1)
        throw new IOException("Exception Message1");
     else
        throw new ClassNotFoundException("Exception Message2");
  } 
} 

class Demo{ 
  public static void main(String args[]){ 
   try{ 
     complete obj=new complete(); 
     obj.mymethod(1); 
   }catch(Exception ex){
     System.out.println(ex);
    } 
  }
}